resolwe/flow/filters.py

```diff
@@ -189,6 +196,12 @@ class Meta(BaseResolweFilter.Meta):
             'started': DATETIME_LOOKUPS[:],
         }}
 
+    def get_always_allowed_arguments(self):
```

Comment:
This is unnecessary, as `hydrate_data` is never used on data endpoint.

Reply:
OK, but why this:
https://github.com/genialis/resolwe/blob/master/resolwe/flow/views/data.py#L65

Reply:
It can probably be removed: https://github.com/genialis/resolwe/blob/ddd7bdd887be1615d8dc8ff72508a7fc0c20c5c6/resolwe/flow/serializers/data.py#L110-L134

Reply:
Done.